torch/testing/_internal/common_fsdp.py

```diff
@@ -1028,7 +1028,7 @@ def _test_fsdp_parity(
                 self.assertEqual(param.device, cpu_device)
             fsdp_loss = fsdp_loss.cuda()
         fsdp_unsharded_params = get_full_params(fsdp_model)
-        torch.testing.assert_allclose(ref_loss, fsdp_loss)
+        torch.testing.assert_close(ref_loss, fsdp_loss, check_dtype=False)
```

Comment:
Strict dtype checking is the default for `torch.testing.assert_close`. Are mismatching dtypes ok here?